types/features/features.go

```diff
@@ -0,0 +1,102 @@
+
+	Cgroup   Cgroup   `json:"cgroup"`
+	Seccomp  Seccomp  `json:"seccomp"`
+	Apparmor Apparmor `json:"apparmor"`
+	Selinux  Selinux  `json:"selinux"`
+	Criu     Criu     `json:"criu"`
+
+	// Annotations contains implementation-specific annotation strings,
+	// such as the implementation version, and third-party extensions.
+	Annotations map[string]string `json:"annotations,omitempty"`
+}
+
+// Seccomp represents the "seccomp" field
+type Seccomp struct {
+	// Enabled is true if seccomp support is compiled in
+	Enabled bool `json:"enabled"`
+
+	// Actions is the list of the recognized actions, e.g., "SCMP_ACT_NOTIFY"
+	// Nil value means "unknown", not "no support for any action".
+	Actions []string `json:"actions,omitempty"`
+
+	// Operators is the list of the recognized actions, e.g., "SCMP_CMP_NE"
+	// Nil value means "unknown", not "no support for any operator".
+	Operators []string `json:"operators,omitempty"`
+
+	// Operators is the list of the recognized archs, e.g., "SCMP_ARCH_X86_64"
+	// Nil value means "unknown", not "no support for any arch".
+	Archs []string `json:"archs,omitempty"`
+}
+
+// Apparmor represents the "apparmor" field
+type Apparmor struct {
+	// Enabled is true if AppArmor support is compiled in
+	// Unrelated to whether the host supports AppArmor or not.
+	// Always true in the current version of runc.
+	Enabled bool `json:"enabled"`
+}
+
+// Selinux represents the "selinux" field
+type Selinux struct {
+	// Enabled is true if SELinux support is compiled in
+	// Unrelated to whether the host supports SELinux or not.
+	// Always true in the current version of runc.
+	Enabled bool `json:"enabled"`
+}
+
+// Criu represents the "criu" field
+type Criu struct {
+	// Enabled is true if CRIU support is compiled in
+	// Unrelated to whether the host supports CRIU or not.
+	// Always true in the current version of runc.
+	Enabled bool `json:"enabled"`
+}
+
+// Cgroup represents the "cgroup" field
+type Cgroup struct {
+	// V1 represents whether Cgroup v1 support is compiled in.
+	// Unrelated to whether the host uses cgroup v1 or not.
+	// Always true in the current version of runc.
+	V1 bool `json:"v1"`
+
+	// V2 represents whether Cgroup v2 support is compiled in.
+	// Unrelated to whether the host uses cgroup v2 or not.
+	// Always true in the current version of runc.
+	V2 bool `json:"v2"`
+
+	// Systemd represents whether systemd-cgroup support is compiled in.
+	// Unrelated to whether the host uses systemd or not.
+	// Always true in the current version of runc.
+	Systemd bool `json:"systemd"`
+
+	// SystemdUser represents whether user-scoped systemd-cgroup support is compiled in.
+	// Unrelated to whether the host uses systemd or not.
+	// Always true in the current version of runc.
+	SystemdUser bool `json:"systemdUser"`
+}
+
+const (
+	// AnnotationRuncVersion represents the version of runc, e.g., "1.2.3", or "1.2.3+dev".
+	// Third party implementations such as crun and runsc SHOULD NOT use this annotation.
```

Comment:
(On second thought, crun MAY use this  annotation to report the most compatible runc version’

Reply:
Reworded: `Third party implementations such as crun and runsc MAY use this annotation to report the most compatible runc version, however, parsing this annotation value is discouraged.`